src/_quick_and_dirty_interactive_testing.py

```diff
@@ -0,0 +1,280 @@
+'''
+PorQua : a python library for portfolio optimization and backtesting
+PorQua is part of GeomScale project
+
+Copyright (c) 2024 Cyril Bachelard
+
+Licensed under GNU LGPL.3, see LICENCE file
+'''
+
+
+############################################################################
+### QUICK-AND-DIRTY TESTING IN INTERACTIVE MODE DURING DEVELOPMENT
```

Comment:
Maybe a better place for this file is in test directory.

Reply:
I would move it to examples, since this is a quick-and-dirty example of how one could run the (additional) code.

Reply:
Sure, examples are fine.